test/SlowTests/SlowTests.csproj

```diff
@@ -103,6 +103,7 @@
     <ProjectReference Include="..\..\src\Voron\Voron.csproj" />
     <ProjectReference Include="..\..\tools\Raven.Migrator\Raven.Migrator.csproj" />
     <ProjectReference Include="..\..\tools\Voron.Recovery\Voron.Recovery.csproj" />
+    <ProjectReference Include="..\EmbeddedTests\EmbeddedTests.csproj" />
```

Comment:
Why is this needed?

Reply:
I reused a ravendb dump creating method from there
https://github.com/ravendb/ravendb/pull/10564/files#diff-d7adabe65d19221ae0f622e52c1b2751R552